evalml/tests/component_tests/test_stacked_ensemble_classifier.py

```diff
@@ -3,170 +3,177 @@
+    assert ProblemTypes.BINARY in StackedEnsembleClassifier.supported_problem_types
+    assert ProblemTypes.MULTICLASS in StackedEnsembleClassifier.supported_problem_types
+    assert StackedEnsembleClassifier.supported_problem_types == [
+        ProblemTypes.BINARY,
+        ProblemTypes.MULTICLASS,
+        ProblemTypes.TIME_SERIES_BINARY,
+        ProblemTypes.TIME_SERIES_MULTICLASS,
     ]
-    clf = SklearnStackedEnsembleClassifier(
-        input_pipelines=input_pipelines, random_seed=5, n_jobs=1
+
+
+def test_stacked_ensemble_nondefault_y():
+    pytest.importorskip(
+        "imblearn.over_sampling",
+        reason="Skipping nondefault y test because imblearn not installed",
     )
-    estimators_used_in_ensemble = mock_stack.call_args[1]["estimators"]
-    assert clf.random_seed == 5
-    assert estimators_used_in_ensemble[0][1].pipeline.random_seed == 3
-    assert estimators_used_in_ensemble[1][1].pipeline.random_seed == 4
-
-
-def test_stacked_ensemble_multilevel(logistic_regression_binary_pipeline_class):
-    # checks passing a stacked ensemble classifier as a final estimator
-    X = pd.DataFrame(np.random.rand(50, 5))
-    y = pd.Series([1, 0] * 25)
-    base = SklearnStackedEnsembleClassifier(
-        input_pipelines=[logistic_regression_binary_pipeline_class(parameters={})],
-        n_jobs=1,
+    X, y = datasets.make_classification(
+        n_samples=100, n_features=20, weights={0: 0.1, 1: 0.9}, random_state=0
     )
-    clf = SklearnStackedEnsembleClassifier(
-        input_pipelines=[logistic_regression_binary_pipeline_class(parameters={})],
-        final_estimator=base,
-        n_jobs=1,
+    input_pipelines = [
+        BinaryClassificationPipeline(
+            {
+                "OS": ["Oversampler", "X", "y"],
+                "rf": [RandomForestClassifier, "OS.x", "OS.y"],
+            },
+            parameters={"OS": {"sampling_ratio": 0.5}},
+        ),
+        BinaryClassificationPipeline(
+            {
+                "OS": ["Oversampler", "X", "y"],
+                "rf": [RandomForestClassifier, "OS.x", "OS.y"],
+            },
+            parameters={
+                "OS": {"sampling_ratio": 0.5},
+                "Random Forest Classifier": {"n_estimators": 22},
+            },
+        ),
+    ]
+
+    pl = _make_stacked_ensemble_pipeline(
+        input_pipelines=input_pipelines,
+        problem_type=ProblemTypes.BINARY,
     )
-    clf.fit(X, y)
-    y_pred = clf.predict(X)
+    pl.fit(X, y)
+    y_pred = pl.predict(X)
     assert len(y_pred) == len(y)
     assert not np.isnan(y_pred).all()
 
 
-def test_stacked_problem_types():
+def test_stacked_ensemble_keep_estimator_parameters(X_y_binary):
+    pytest.importorskip(
+        "imblearn.over_sampling",
+        reason="Skipping nondefault y test because imblearn not installed",
+    )
+    X, y = X_y_binary
+    input_pipelines = [
+        BinaryClassificationPipeline(
+            {
+                "OS": ["Oversampler", "X", "y"],
```

Comment:
Omega nitpick but if we're just testing that the ensemble parameters are kept, we don't need to use the oversampler which only runs when it's imported right? We can use any of our other components? :)

Reply:
True, I'll update this to use an imputer instead!